uws.js

```diff
@@ -17,7 +17,7 @@
 
 module.exports = (() => {
 	try {
-		return require('./uws_' + process.platform + '_' + process.arch + '_' + process.versions.modules + '.node');
+		return require('@uws/uws_' + process.platform + '_' + process.arch + '_' + process.versions.modules);
```

Comment:
This is the new path that resolves to the specific binary installed as its own package